### PR TITLE
Lint and build fixes

### PR DESCRIPTION
**Pull request by jbranson000**
https://github.com/adobe/xdm/issues/1017

### CODE REVIEW

File: components/datatypes/transaction.schema.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/transaction",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Transaction",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:industry": [
```
Comment (mjasrotia) [inline components/datatypes/transaction.schema.json:13]: We could also add "retail" here right ? A sales transaction is a common thing in retail Industry.

File: components/mixins/experience-event/experienceevent-bill-pay-steps.schema.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:industry": ["financial_services"],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Bill pay steps such as start, save, failure, and complete.",
+  "definitions": {
+    "billPaySteps": {
+      "properties": {
+        "xdm:financialServices": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-bill-pay-steps.schema.json:20]: This was supposed to be "personalFinance" right ?

**Comment by mjasrotia** [comment]
@jbranson000  other than couple comments I have put in, the PR looks good to me. Please see those comments.

File: components/mixins/experience-event/experienceevent-bill-pay-steps.schema.json (hunk 2 above)
Comment (jbranson000) [reply components/mixins/experience-event/experienceevent-bill-pay-steps.schema.json]: Yes - it was out of sync locally with the last push.  Fixed now.

File: components/datatypes/transaction.schema.json (hunk 1 above)
Comment (jbranson000) [reply components/datatypes/transaction.schema.json:13]: Yes, I've added that.

**Review by mjasrotia — APPROVED** [review]

File: components/mixins/experience-event/experienceevent-bill-pay-steps.schema.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  "meta:industry": ["financial_services"],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Bill pay steps such as start, save, failure, and complete.",
+  "definitions": {
+    "billPaySteps": {
+      "properties": {
+        "xdm:personalFinance": {
+          "type": "object",
+          "properties": {
+            "xdm:billPaySteps": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-bill-pay-steps.schema.json:23]: Please add type and properties under this.

**Review by prabhum2 — APPROVED** [review]

**Pull request merged**